### ISSUE TITLE
Fix the Online Tests

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#1278]
Apart from a few other issues, it seems that the GOES URLs have changed.

https://travis-ci.org/sunpy/sunpy/jobs/48367960#L1365

**Comment by bsipocz** [comment]
It seems that the number of failing online tests are growing, currently I got 13:

```
sunpy/database/tests/test_attrs.py ................................
sunpy/database/tests/test_caching.py ....
sunpy/database/tests/test_commands.py ...........................
sunpy/database/tests/test_database.py ...............................F.FFF................................FF..FFF..
sunpy/database/tests/test_serialize.py ..........................
sunpy/database/tests/test_table.txt s
sunpy/database/tests/test_tables.py ......FE......F.F....
sunpy/image/tests/test_coalignment.py .............
sunpy/image/tests/test_resample.py .....
sunpy/image/tests/test_transform.py ...................................
sunpy/instr/tests/test_aia.py ..
sunpy/instr/tests/test_fermi.py ..
sunpy/instr/tests/test_goes.py .......................
sunpy/instr/tests/test_iris.py ..
sunpy/instr/tests/test_lyra.py .........
sunpy/io/tests/test_ana.py ......
sunpy/io/tests/test_filetools.py ..s.s....
sunpy/io/tests/test_fits.py ........
sunpy/io/tests/test_jp2.py sss
sunpy/lightcurve/tests/test_eve.py ...
sunpy/lightcurve/tests/test_goes.py .........
sunpy/lightcurve/tests/test_lightcurve.py ........
sunpy/lightcurve/tests/test_lyra.py ....
sunpy/lightcurve/tests/test_noaa.py ............
sunpy/lightcurve/tests/test_norh.py .
sunpy/lightcurve/tests/test_rhessi.py ........
sunpy/map/sources/tests/test_aia_source.py ....
sunpy/map/sources/tests/test_cor_source.py ....
sunpy/map/sources/tests/test_eit_source.py ....................................................
sunpy/map/sources/tests/test_euvi_source.py ....
sunpy/map/sources/tests/test_hi_source.py ....
sunpy/map/sources/tests/test_hmi_source.py ....
sunpy/map/sources/tests/test_iris_source.py ...
sunpy/map/sources/tests/test_lasco_source.py ....
sunpy/map/sources/tests/test_mdi_source.py ....
sunpy/map/sources/tests/test_sot_source.py .......
sunpy/map/sources/tests/test_swap_source.py ................
sunpy/map/sources/tests/test_xrt_source.py .....
sunpy/map/tests/test_header.py .
sunpy/map/tests/test_map_factory.py ...........
sunpy/map/tests/test_mapbase.py ...................................................
sunpy/map/tests/test_mapcube.py ...
sunpy/net/jsoc/tests/test_attr.py ...........
sunpy/net/jsoc/tests/test_jsoc.py .....................
sunpy/net/tests/test_attr.py .
sunpy/net/tests/test_download.py ....
sunpy/net/tests/test_hek.py ............
sunpy/net/tests/test_hek2vso.py ..
sunpy/net/tests/test_helio.py ..
sunpy/net/tests/test_util.py ...
sunpy/net/tests/test_vso.py ....................
sunpy/physics/tests/test_differential_rotation.py ...........
sunpy/physics/tests/test_solar_rotation.py ..
sunpy/roi/tests/test_chaincode.py ...........
sunpy/roi/tests/test_roi.py .........
sunpy/spectra/tests/test_callisto.py ..XX.................
sunpy/spectra/tests/test_spectrogram.py .....................................
sunpy/sun/tests/test_sun.py ................
sunpy/tests/tests/test_helpers.py .
sunpy/tests/tests/test_main.py .........
sunpy/time/tests/test_julian.py ...............
sunpy/time/tests/test_time.py ............
sunpy/time/tests/test_timerange.py ..........................
sunpy/util/tests/test_cond_dispatch.py ......
sunpy/util/tests/test_datatype_factory_base.py ....
sunpy/util/tests/test_multimethod.py ..
sunpy/util/tests/test_scraper.py ..............F
sunpy/util/tests/test_sysinfo.py .
sunpy/util/tests/test_util.py ............
sunpy/util/tests/test_xml.py ....................
sunpy/wcs/tests/test_wcs.py .........
```

**Comment by Cadair** [comment]
closed by #1902
